### ISSUE TITLE
MVP of financial reports export

### ISSUE THREAD

**Issue by jel535s** [MidwestAccessCoalition/lockbox_rails#351]
## Context
As the finance team at MAC, we want to be able to access financial reporting from the Lockbox app.  Here are the current asks:

Export type:
Either an excel workbook or an airtable,   um, table
 
Data set:
every Lockbox action, however with each line item of a Support Request broken out as a separate line in the dataset
 
Columns/filters:
Action type: Client support request, cash add, or reconciliation
Status: Pending/Cancelled/Completed
Partner
Client Reference ID
Date submitted (support request created)
Date of expense (COMPLETED date)
Support Request breakdown category/Expense category (should match Quickbooks, blocked by #395) (or NA if this is a Reconciliation or Cash Add)
Amount
Running total balance by date- Balance calculation for this should treat pending requests as NOT completed (likely blocked by #388)
Sum of all lockbox balances where pending is NOT being treated as completed (as calculated by #388)
 
Views built in as separate tabs, all with the above columns/filters, as relevant:
All lockbox actions across all partners
Client Expenses Only (no cash adds or   reconciliations) - COMPLETED Activity plus balance report per clinic --   IGNORE PENDING + CANCELED
Client Expenses Only (no cash adds or   reconciliations) - Only COMPLETED Support Requests -- IGNORE PENDING +   CANCELED- every line item sorted by categories

## Process
Open to interpretation-possibly we can use Heroku's dataclips feature as Nicole mentions below

## Acceptance Criteria/Definition of Done
MVP -- the ability for the MAC tech team to export all lockbox transactions for a given lockbox to a spreadsheet or airtable upon request.  In future version as built out in #390 this will be something finance can pull themselves- this is just the MVP.

**Comment by BintLopez** [comment]
Heroku has a cool dataclips feature where we can share views of specific sql queries. That might be a good workaround for financial reporting until we can build out a dashboard in the app

**Comment by jel535s** [comment]
Ver2 is currently blocked/superseded by #390

**Comment by nsiwnf** [comment]
Assigning this to myself to work on the backend for this since that part is spec'd out. Holding off on any UX changes until theres specs for that too

**Comment by gujacq** [comment]
@jel535s and @BintLopez let's talk about this ticket either on our sync Friday or at Sunday's tech meeting. I feel like I need a refresher on this

**Comment by gujacq** [comment]
Decision from team - MVP will be a button to export to a CSV file

**Comment by gujacq** [comment]
Sree is building out backend functionality right now

Jacqui and Jesse to talk through UI for display

**Comment by gujacq** [comment]
@nsiwnf Let's add a link called "Financial Export" to this admin screen under the "Quick Actions" column on the right side: https://invis.io/9EMYOUG7ZBA#/408360708_dashboard_-_MAC_Admin_-menu_Open-
